### DISCUSSION TITLE
Q: How I can get repository, tags and digest from the target by name?

### DISCUSSION THREAD

**Discussion by farcop** [bazel-contrib/rules_oci#330] [Q&A] [answered]
I am trying to find the way to get repository, tags and digest from the target by target name from oci_push targets that are scattered across my workspace.

For this with rules_docker I used provider PushInfo just like this:
```
for d in ctx.attr.deps:
        if PushInfo in d:
            pi = d[PushInfo]
            input_files.append(pi.digest)
            image = pi.registry
            if image != "": image += "/"
            image += pi.repository + ":" + pi.tag
```
Is it possible to do something like this with rules_oci ?

**Comment by farcop** [comment]
It must be very close to https://github.com/bazel-contrib/rules_oci/issues/60

**Comment by thesayyn** [comment, marked as answer]
You can get this information from the stdout of oci_push. oci_push don't have a PushInfo provider.

**Reply by farcop** [reply]
Sorry but I can't understand how to do it. This is not answer
